static/app/views/performance/traces/tracesSearchBar.tsx

```diff
@@ -1,22 +1,99 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
 import SearchBar from 'sentry/components/events/searchBar';
-import type {SmartSearchBarProps} from 'sentry/components/smartSearchBar';
+import {IconAdd, IconClose} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
 import useOrganization from 'sentry/utils/useOrganization';
 
 interface TracesSearchBarProps {
-  handleSearch: SmartSearchBarProps['onSearch'];
-  query: string;
+  handleClearSearch: (index: number) => boolean;
+  handleSearch: (index: number, query: string) => void;
+  queries: string[];
 }
 
-export function TracesSearchBar({query, handleSearch}: TracesSearchBarProps) {
+const getSpanName = (index: number) => {
+  const spanNames = [t('Span A'), t('Span B'), t('Span C')];
```

Comment:
Nit: move this into a constant outside the function

Reply:
In that case the translate function would run during import, which isn't a good tradeoff. Hiding consts in helpers is fine.